pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -136,6 +135,6 @@ rustPlatform.buildRustPackage rec {
     description = "GPU-accelerated cross-platform terminal emulator and multiplexer written by @wez and implemented in Rust";
     homepage = "https://wezfurlong.org/wezterm";
     license = licenses.mit;
-    maintainers = with maintainers; [ SuperSandro2000 ];
+    maintainers = with maintainers; [ SuperSandro2000 mimame ];
```

Comment:
Do you have a darwin box?

Reply:
Yes, why?

Reply:
thats good, then I can ignore darwin even more